src/index.js

```diff
@@ -1,93 +1,18 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+import express from 'express';
+import cors from 'cors';
 
 // Create global app object
 const app = express();
 
 app.use(cors());
 
 // Normal express config defaults
-app.use(require("morgan")("dev"));
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
-
-app.use(require("method-override")());
-app.use(express.static(__dirname + "/public"));
-
-app.use(
-    session({
-        secret: "authorshaven",
-        cookie: { maxAge: 60000 },
-        resave: false,
-        saveUninitialized: false
-    })
-);
-
-if (!isProduction) {
-    app.use(errorhandler());
-}
-
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
-
-require("./models/User");
-
-app.use(require("./routes"));
-
-/// catch 404 and forward to error handler
-app.use(function(req, res, next) {
-    const err = new Error("Not Found");
-    err.status = 404;
-    next(err);
-});
-
-/// error handlers
-
-// development error handler
-// will print stacktrace
-if (!isProduction) {
-    app.use(function(err, req, res, next) {
-        console.log(err.stack);
-
-        res.status(err.status || 500);
-
-        res.json({
-            errors: {
-                message: err.message,
-                error: err
-            }
-        });
-    });
-}
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
 
-// production error handler
-// no stacktraces leaked to user
-app.use(function(err, req, res, next) {
-    res.status(err.status || 500);
-    res.json({
-        errors: {
-            message: err.message,
-            error: {}
-        }
-    });
-});
+app.get('/', (req, res) => res.json({ status: res.statusCode, message: 'Welcome to Wolfsbane' }));
```

Comment:
Nice work bro, I think you should have set the status code directly as in
res.status(200).json({ status: 200, message: 'welcome to wolfsbane'})
doing this is more explicit, allowing us to know the actual status code while sending the response

Reply:
@Oliver-ke, for now, we can leave it the way it is. We'll need to improve the welcome page later.